python/xorbits/_mars/storage/filesystem.py

```diff
@@ -149,3 +149,59 @@ class DiskStorage(FileSystemStorage):
     async def setup(cls, **kwargs) -> Tuple[Dict, Dict]:
         kwargs["level"] = StorageLevel.DISK
         return await super().setup(**kwargs)
+
+
+@register_storage_backend
+class AlluxioStorage(FileSystemStorage):
+    name = "alluxio"
+
+    def __init__(
+        self,
+        root_dirs: List[str],
+        local_environ: bool,
+        fs: FileSystem = None,
+        level: StorageLevel = None,
+        size: int = None,
```

Comment:
If multiple directories is not allowed, `root_dirs` could be `root_dir`, also `fs` is not needed as it is always a `LocalFileSystem`.